source/MRMesh/MROffset.h

```diff
@@ -30,6 +30,10 @@ struct OffsetParameters : BaseShellParameters
 
     /// defines particular implementation of IFastWindingNumber interface that will compute windings. If it is not specified, default FastWindingNumber is used
     std::shared_ptr<IFastWindingNumber> fwn;
+
+    /// use FunctionVolume for voxel grid representation; decreases memory consumption but increases computation time
+    /// used only by mcOffsetMesh and sharpOffsetMesh methods
```

Comment:
Better give some approximate numbers: how much memory is saved in % and how much CPU is necessary